src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala

```diff
@@ -350,4 +350,111 @@ class AzulZuluMigrations {
     )
   }
 
+  @ChangeSet(
+    order = "012",
+    id = "0012-add-add_zulu_11_0_8",
+    author = "poad"
+  )
+  def migrate012(implicit db: MongoDatabase) =
+    Map(
+      Linux64 -> ("zulu11.41.23-ca-jdk11.0.8-linux_x64.tar.gz"),
```

Comment:
No need for tuples, you can simply compose the map with entries like: `Linux64 -> "xxx.tar.gz"`